src/SDKs/Peering/Management.Peering/Microsoft.Azure.Management.Peering.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
```

Comment:
Please take a look at a csproj [here](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj) and remove any additional propertygroups/itemgroups